projects/graphql-client/src/graphql-apollo-integration.test.ts

```diff
@@ -24,14 +24,23 @@ describe('GraphQl Apollo Integration Service', () => {
     ]
   });
 
-  const buildRequestString = (rootPath: string = 'test'): string =>
-    new GraphQlRequestBuilder().withSelects({ path: rootPath, children: [{ path: 'id' }, { path: 'value' }] }).build();
+  const buildRequestString = (rootPath: string = 'test', params: string[] = []): string =>
+    new GraphQlRequestBuilder()
+      .withSelects({
+        path: rootPath,
+        children: [{ path: 'id' }, { path: 'value' }],
+        ...params.map(param => ({ path: param }))
```

Comment:
should these be inside the children array? or am I misreading it

Reply:
aah.. I added this for the other test which i removed https://github.com/hypertrace/hypertrace-ui/pull/223#discussion_r499072296

I can remove this depending up your comments on the other comment.

Reply:
should be fixed.